dot/peerset/peerset.go

```diff
@@ -183,6 +183,8 @@ func NewConfigSet(in, out uint32, reservedOnly bool, allocTime time.Duration) *C
 	}
 
 	return &ConfigSet{
+		// Why are we using an array of config in the set, when we are
+		// using just one config
```

Comment:
#1886 @arijitAD could you give more context on this?

if I remember correctly it was so we could score the different stream protocols differently (eg. sync, block-announce, etc)